Code/RDBoost/Wrap/RDBase.cpp

```diff
@@ -78,9 +78,11 @@ struct PyLogStream : std::ostream, std::streambuf {
   }
 
   ~PyLogStream() {
+    #if PY_VERSION_HEX > 0x03070000
```

Comment:
I think the rest of the code base has the # on the first character.

I’m sadly old enough when this was required.  :)

Reply:
Oh, I hadn’t noticed! I’ll fix it straightaway.